server.cpp

```diff
@@ -0,0 +1,117 @@
+#include "server.hpp"
+#include <iostream>
+
+const int ChatServer::serverPort = 54000;
```

Comment:
If the value is hard coded, then go to the class definition in the header file, and write 
'static constexpr int serverPort = 54000' 
To let the compiler evaluate it during compilation time